test/lib/ufe/testSelection.py

```diff
@@ -66,13 +66,14 @@ def runTestSelection(self, selectCmd):
         # time, so we select 6 different items, one at a time.
         shapeSegment = mayaUtils.createUfePathSegment(
             "|world|mayaUsdProxy1|mayaUsdProxyShape1")
-        names = ["pCube1", "pCylinder1", "pSphere1"]
+        ufeNames = ["cubeXform", "cylinderXform", "sphereXform"]
+        mayaNames = ["pCube1", "pCylinder1", "pSphere1"]
         usdPaths = []
-        for n in ["/" + o for o in names]:
+        for n in ["/" + o for o in ufeNames]:
             usdPaths.append(ufe.Path(
                 [shapeSegment, usdUtils.createUfePathSegment(n)]))
         mayaPaths = []
-        for n in ["|world|" + o for o in names]:
+        for n in ["|world|" + o for o in mayaNames]:
             mayaPaths.append(ufe.Path(mayaUtils.createUfePathSegment(n)))
```

Comment:
@ppt-adsk Adopt to the recent changes in simpleSceneTRS:  Separated "maya" and "ufe" names list.